Added panel actions menu content (#77162)
grafana/grafana: docs/sources/panels-visualizations/panel-editor-overview/index.md

![Panel editor](/media/docs/grafana/panels-visualizations/screenshot-panel-editor-view.png)

To add a panel in a new dashboard click **+ Add visualization** in the middle of the dashboard. To add a panel to an existing dashboard, click **Add** in the dashboard header and select **Visualization** in the drop-down:

![Add dropdown](/media/docs/grafana/dashboards/screenshot-add-dropdown-10.0.png)

> ## Panel menu

Reviewer: Mainly here to review the Extensions part, but I'd question adding this in a section about the Panel editor overview - since its not part of the editor... I would either have a new page for Panel menu or, if the focus is on enabling users to discover the alerting addition, just to ensure the alerting docs are updated (which they may already be).

Author: Thanks for the quick review! 

The alerting docs are already updated so this was a quick and dirty way (given the time crunch) to get something up for the purpose of having a place from which to link to the alerting docs.

While the panel menu isn't specifically part of the panel editor, I'm not sure from a user perspective that there's as much of a distinction as we might feel like there is internally. I definitely don't think this warrants an entire page on its own. I'd like to get an opinion from the Dashboards team as well though. @dprokop?

@Ijin08 - from a UX perspective, do you have thoughts on this?

Author: One thing I can do is add some connective tissue. Something like "You can access the panel editor from the panel menu. The panel menu also includes these other options"

Author: @sympatheticmoose - @Ijin08 and I talked through placement of this content and we decided that there's a need for a panel overview page that doesn't currently exist. There's not enough time for me to do that before I go on PTO, so we'll go with this treatment for now, but I'll make the change later.

Reviewer: sounds good to me 👍

> ## Panel actions menu

Author: ```suggestion
## Panel menu
```

> When you hover over the top-right corner of any panel, the panel menu icon appears. Click the icon to access the following actions:

Author: ```suggestion
To access the panel editor, hover over the top-right corner of any panel. Click the panel menu icon that appears and select **Edit**. The panel menu gives you access to the following actions:
```

> When you hover over the top-right corner of any panel, the actions menu appears. Click the menu icon to access the following actions:

Author: ```suggestion
When you hover over the top-right corner of any panel, the menu icon appears. Click the menu icon to access the following actions:
```

- **View**: View the panel in full screen.
- **Edit**: Open the panel editor to edit panel and visualization options.
- **Share**: Share the panel as a link, embed, or library panel.
- **Explore**: Open the panel in **Explore**, where you can focus on your query.
- **Inspect**: Open the **Inspect** drawer, where you can review the panel data, stats, metadata, JSON, and query.
  - **Data**: Open the **Inspect** drawer in the **Data** tab.
  - **Query**: Open the **Inspect** drawer in the **Query** tab.
  - **Panel JSON**: Open the **Inspect** drawer in the **JSON** tab.

> - **Extensions**: Access installed extensions that let you quickly... such as, declaring an incident.

Author: ```suggestion
- **Extensions**: Access other actions that have been added using [plugins](https://grafana.com/grafana/plugins/) such as, declaring an incident (enabled by default in Grafana Cloud).
```

> - **Extensions**: Access other actions that have been added using [plugins](https://grafana.com/grafana/plugins/) such as, declaring an incident (enabled by default in Grafana Cloud).

Author: ```suggestion
- **Extensions**: Access other actions that are added using UI extensions, such as declaring an incident. Note that this option doesn't appear unless you have app plugins installed. Learn more in the [UI extensions documentation](https://grafana.com/developers/plugin-tools/ui-extensions/).
```

> - **Extensions**: Access other actions that are added using UI extensions, such as declaring an incident. Note that this option doesn't appear unless you have app plugins installed. Learn more in the [UI extensions documentation](https://grafana.com/developers/plugin-tools/ui-extensions/).

Reviewer: ```suggestion
- **Extensions**: Access other actions provided by installed applications, such as declaring an incident. Note that this option doesn't appear unless you have app plugins installed which contribute an [extension](https://grafana.com/developers/plugin-tools/ui-extensions/) to the panel menu.
```

> - **Extensions**: Access other actions provided by installed applications, such as declaring an incident. Note that this option doesn't appear unless you have app plugins installed which contribute an [extension](https://grafana.com/developers/plugin-tools/ui-extensions/) to the panel menu.

Reviewer 2: This looks great to me!

- **More**: Access other panel actions.
  - **Duplicate**: Make a copy of the panel. Duplicated panels query data separately from the original panel. You can use the special `Dashboard` data source to [share the same query results across panels][] instead.
  - **Copy**: Copy the panel to the clipboard.
  - **Create library panel**: Create a panel that can be imported into other dashboards.

>   - **Create alert**: [Create an alert][] directly from the panel.

Reviewer 3: To me it reads like I can create an alert inline. I would make it clear for the user that the they will leave the dashboard view.

>   - **Create alert**: [Create an alert][] directly from the panel.

Author: ```suggestion
  - **Create alert**: Open the alert rule configuration page in **Alerting**, where you can [create a Grafana-managed alert] based on the panel. 
```

![Add dropdown](/media/docs/grafana/dashboards/screenshot-add-dropdown-10.0.png)

> Panel actions menu

Author: ```suggestion
## Panel actions menu
```

[the overview of Grafana Alerting]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/alerting"
[the overview of Grafana Alerting]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/alerting"

[share the same query results across panels]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/panels-visualizations/query-transform-data/share-query"
[share the same query results across panels]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/visualizations/panels-visualizations/query-transform-data/share-query"

> [Create an alert]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/alerting/alerting-rules/create-grafana-managed-rule/#create-alerts-from-panels"
> [Create an alert]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/alerting-and-irm/alerting/alerting-rules/create-grafana-managed-rule#create-alerts-from-panels"

Author: ```suggestion
[create a Grafana-managed alert]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/alerting/alerting-rules/create-grafana-managed-rule/#create-alerts-from-panels"
[create a Grafana-managed alert]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/alerting-and-irm/alerting/alerting-rules/create-grafana-managed-rule#create-alerts-from-panels"
```
